tests/templates/kuttl/keycloak-user-info/02-install-keycloak.yaml.j2

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - << EOF
+      ---
+      apiVersion: secrets.stackable.tech/v1alpha1
+      kind: SecretClass
+      metadata:
+        name: keycloak-tls-$NAMESPACE
+      spec:
+        backend:
+          autoTls:
+            ca:
+              autoGenerate: true
+              secret:
+                name: keycloak-tls-ca-$NAMESPACE
+                namespace: $NAMESPACE
```

Comment:
Do we really need to stub out the default vanilla `tls` secretclass?

Reply:
Agreed, this ensures TLS is correctly trusted, and not by accident.

Reply:
Sure.